packages/alchemy/src/react/components/auth/card/index.tsx

```diff
@@ -48,7 +56,10 @@ export const AuthCard = (props: AuthCardProps) => {
         setAuthStep,
       }}
     >
-      <div className="modal-box flex flex-col items-center gap-5">
+      <div className="modal-box relative flex flex-col items-center gap-5 text-fg-primary">
+        {props.showNavigation && (
+          <Navigation showingBack={false} onClose={closeAuthModal} />
+        )}
```

Comment:
oh wait should this not be in `packages/alchemy/src/react/components/auth/modal.tsx`?

Reply:
discussed offline 🤝